_includes/signup.html

```diff
@@ -1,7 +1,5 @@
-<div class="sign-up-container">
-  <a href="https://runnable.com/signup/" class="well well-anchor sign-up-box grid-block col track-sign-up noscroll">
-    <img class="img no-touching" src="{{ site.url }}/images/runnabear-hello.png">
-    <p class="p">Try Runnable! It’s <strong class="weight-strong">free</strong> for 14 days.
-    <div data-target="#sign-up" class="btn btn-sm green js-modal track-sign-up">Sign Up</div>
-  </a>
-</div>
+<a href="https://runnable.com/signup/" class="sign-up-box grid-block col well well-anchor track-sign-up">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7440805/25594419/4f888930-2e75-11e7-926c-d11d713549a1.png)
The text on this element looks a bit too light. Also it looks like it's missing some margin between the "Sign Up" button and the text (compared to the version on runnable.com)